cmd/korg/utils.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"k8s.io/test-infra/prow/config/org"
+	"sigs.k8s.io/yaml"
+)
+
+func stringInSlice(slice []string, key string) bool {
+	for _, e := range slice {
+		if key == e {
+			return true
+		}
+	}
+
+	return false
+}
+
+func validateOrgs(orgs []string) (valid []string, invalid []string, invalidPresent bool) {
```

Comment:
Is there a reason why we return the `valid` orgs? It looks like they aren't being used anywhere. :thinking:

Reply:
Simplified the implementation now. Valid orgs are not returned anymore.